uiServices/src/services.go

```diff
@@ -96,5 +94,141 @@ func validateJsonWithSchema(config []byte) bool {
         }
     }
     return true
+}
+
+func getConfigs(rw http.ResponseWriter, req *http.Request){
+
+    configPathDir := req.Header.Get("configPathDir")
+    configName := chi.URLParam(req, "configName")
+
+    if !strings.HasSuffix(configPathDir, "/"){
+        configPathDir = configPathDir + "/"
+    }
+
+    file, err := os.Open(fmt.Sprintf("%s%s.xml", configPathDir, configName))
+    if len(configPathDir) <= 1{
```

Comment:
move the configPathDir length check to before you try to open the file. Note: I flagged this in https://github.com/xtracdev/automated-perf-test/pull/40 , so you can make the change there and pull it in to this branch.